tabs/osd.js

```diff
@@ -340,10 +342,9 @@ OSD.updateDisplaySize = function() {
 OSD.msp = {
   encodeOther: function() {
     var result = [-1, OSD.data.video_system];
-    if (semver.gte(CONFIG.apiVersion, "1.21.0")) {
+    if (semver.gte(CONFIG.flightControllerVersion, "3.0.1")) {
```

Comment:
>  Thing is that cleanflight also uses the same api versions, while MSP is different.

I thought @martinbudden was working on getting that fixed?

I thought we'd agreed that we'd switch when `msp.js` / `MSPHelper.js` was switched over. Have you started the work to do this?